app/templates/custom-elements/remote-screen.html

```diff
@@ -77,6 +83,24 @@
           screenImg.addEventListener("contextmenu", (evt) => {
             evt.preventDefault();
           });
+          let isTouchScreen = false;
+          this.shadowRoot.addEventListener("touchend", () => {
+            isTouchScreen = true;
+          });
+          this.shadowRoot.addEventListener("click", () => {
+            if (isTouchScreen) {
+              this.shadowRoot.getElementById("mobile-keyboard-input").focus();
+            }
+          });
+          this.shadowRoot
+            .getElementById("mobile-keyboard-input")
+            .addEventListener("input", (evt) => {
+              if (evt.target.style.display != "none") {
```

Comment:
It doesn't look like the if clause is necessary since we never set display to `none`.